neovim/plugin/decorators.py

```diff
@@ -22,7 +22,7 @@ def plugin(cls):
     # decorated functions have a bound Nvim instance as first argument.
     # For methods in a plugin-decorated class this is not required, because
     # the class initializer will already receive the nvim object.
-    predicate = lambda fn: hasattr(fn, '_nvim_bind')
+    predicate = lambda fn: hasattr(fn, '_nvim_bind')  # noqa: E731
```

Comment:
This just makes the code ugly for no reason. If we think named lambda is OK here, why would it be wrong somewhere else?

Reply:
I've thought it's a special case here for some reason, but not really read the comment.

So you would like to disable E731 globally then?